src/jmespath.net/JmesPathScanner.cs

```diff
@@ -369,9 +369,16 @@ void RestoreBuffCtx(BufferContext value)
             SetSource(file, 0); // unicode option
         }
 
-        public JmesPathScanner(Stream file, string codepage) {
-            SetSource(file, CodePageHandling.GetCodePage(codepage));
-        }   
+        public JmesPathScanner(Stream file, int codepage)

```

Comment:
This would be better in a separate file, since this file gets overwritten each time the scanner is re-generated.

Reply:
Done :)